backend/src/app.ts

```diff
@@ -48,4 +51,31 @@ app.get('/reviews/:idType/:id', async (req, res) => {
   res.status(200).send(JSON.stringify(reviews));
 });
 
+app.post('/landlords', async (req, res) => {
+  const doc = landlordCollection.doc();
+  const landlord: Landlord = req.body as Landlord;
+  const searchableKeywords: string[] = generateKeywords(landlord.name);
+  const data = { ...landlord, searchableKeywords };
+  doc.set(data);
+  res.status(201).send(doc.id);
+});
+
+app.get('/reviews/:query', async (req, res) => {
```

Comment:
We should use query params instead of route params since it's more idiomatic for this use-case (`/reviews/` and then access through `req.query.something`)